modules/metering-configure-persistentvolumes.adoc

```diff
@@ -3,16 +3,17 @@
 // * metering/configuring_metering/metering-configure-hive-metastore.adoc
 
 [id="metering-configure-persistentvolumes_{context}"]
-= Configuring PersistentVolumes
+= Configuring persistent volumes
 
-By default, Hive requires one PersistentVolume to operate.
+By default, Hive requires one persistent volume to operate.
 
-`hive-metastore-db-data` is the main PersistentVolumeClaim (PVC) required by default. This PVC is used by the Hive metastore to store metadata about tables, such as table name, columns, and location. Hive metastore is used by Presto and the Hive server to look up table metadata when processing queries. You remove this requirement by using MySQL or PostgreSQL for the Hive metastore database.
+`hive-metastore-db-data` is the main persistent volume claim (PVC) required by default. This PVC is used by the Hive metastore to store metadata about tables, such as table name, columns, and location. Hive metastore is used by Presto and the Hive server to look up table metadata when processing queries. You remove this requirement by using MySQL or PostgreSQL for the Hive metastore database.
 
-To install, Hive metastore requires that dynamic volume provisioning be enabled via a StorageClass, a persistent volume of the correct size must be manually pre-created, or that you use a pre-existing MySQL or PostgreSQL database.
+To install, Hive metastore requires that dynamic volume provisioning is enabled in a `StorageClass` object, a persistent volume of the correct size must be manually pre-created, or you use a pre-existing MySQL or PostgreSQL database.
```

Comment:
What's the difference here between resources that get code bracketed vs. ones that don't? I've seen PersistentVolume -> persistent volume, but StorageClass kept the same here.

Reply:
> What's the difference here between resources that get code bracketed vs. ones that don't? I've seen PersistentVolume -> persistent volume, but StorageClass kept the same here.

@timflannagan Good catch. On the rest of this page, it's kept as "storage class". For general references to API objects, we are defaulting to not using code brackets (backticks). I can change this instance to "storage class" because it is not referencing a specific example.

Reply:
That sounds good to me - thanks.